src/main/resources/containers.json

```diff
@@ -1250,8 +1250,6 @@
     ]
   },
   {
-    "artifact_id": "glassfish",
-    "group_id": "org.glassfish.main.distributions",
     "name": "Arquillian Container GlassFish Managed",
```

Comment:
I think we can't get all available properties for container to configure from container adapter which we are maintaining in YAML. I believe each container has some different properties, it not exactly the same.

Good point about If any property changes. But I think the probability of changing any property is very less.